module/evdi_drv.h

```diff
@@ -22,7 +22,11 @@
 #include <drm/drm_crtc_helper.h>
 #include <drm/drm_rect.h>
 #include <drm/drm_gem.h>
+#if KERNEL_VERSION(5, 3, 0) >= LINUX_VERSION_CODE
```

Comment:
Isn't the new situation from 5.4.0? This change breaks build for kernel 5.3.18 at least.